cli/src/main/java/com/devonfw/tools/ide/commandlet/VersionGetCommandlet.java

```diff
@@ -41,7 +39,11 @@ public void run() {
     ToolCommandlet commandlet = this.tool.getValue();
     VersionIdentifier installedVersion = commandlet.getInstalledVersion();
     if (installedVersion == null) {
-      throw new CliException("Tool " + commandlet.getName() + " is not installed!", ProcessResult.TOOL_NOT_INSTALLED);
+      this.context.info("The configured version for tool {} is {}", commandlet.getName(),
+          commandlet.getConfiguredVersion());
+      this.context.info("To install that version call the following command:");
+      this.context.info("ide install {}", commandlet.getName());
+      return;
     }
     this.context.info(installedVersion.toString());
```

Comment:
See my comment in the issue. I would rather do something like this:
```suggestion
      this.context.debug("The configured version for tool {} is:", commandlet.getName());
      this.context.info(commandlet.getConfiguredVersion());
      this.context.debug("To install that version call the following command:");
      this.context.debug("ide install {}", commandlet.getName());
    } else {
      this.context.info(installedVersion.toString());
    }
```